playbooks/libvirt/openshift-cluster/tasks/configure_libvirt_storage_pool.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Create libvirt storage directory for openshift
+  file:
+    dest: "{{ libvirt_storage_pool_path }}"
+    state: directory
+
+- acl:
+    default: yes
+    entity: kvm
+    etype: group
+    name: "{{ libvirt_storage_pool_path }}"
+    permissions: rwx
+    state: present
+
+- name: Test if libvirt storage pool for openshift already exists
+  command: "virsh -c {{ libvirt_uri }} pool-info {{ libvirt_storage_pool }}"
+  register: pool_info_result
+  changed_when: False
+  failed_when: "pool_info_result.rc != 0 and 'error: Storage pool not found:' not in pool_info_result.stderr"
+
+- name: Create the libvirt storage pool for openshift
+  command: 'virsh -c {{ libvirt_uri }} pool-create-as {{ libvirt_storage_pool }} dir --target {{ libvirt_storage_pool_path }}'
+  when: pool_info_result.rc == 1
+
+- name: Refresh the libvirt storage pool for openshift
+  command: 'virsh -c {{ libvirt_uri }} pool-refresh {{ libvirt_storage_pool }}'
```

Comment:
The goal of the `virsh pool-refresh` command is to have `libvirt` rescan the storage pool to discover files/directories that were created/deleted manually (i.e.: without a `virsh vol-XXX` command) after the creation of the pool.

When a pool is created, the content of the path is already scanned.
So, the `pool-refresh` just following the `pool-create` is useless.

See https://github.com/detiber/openshift-ansible/pull/19